tests/unit/script/size_of.rs

```diff
@@ -30,10 +30,10 @@ macro_rules! sizeof_checker (
 
 // Update the sizes here
 sizeof_checker!(size_event_target, EventTarget, 40);
-sizeof_checker!(size_node, Node, 184);
-sizeof_checker!(size_element, Element, 432);
-sizeof_checker!(size_htmlelement, HTMLElement, 448);
-sizeof_checker!(size_div, HTMLDivElement, 448);
-sizeof_checker!(size_span, HTMLSpanElement, 448);
-sizeof_checker!(size_text, Text, 216);
-sizeof_checker!(size_characterdata, CharacterData, 216);
+sizeof_checker!(size_node, Node, 200);
```

Comment:
This looks really unfortunate, we should really have a `RareData`-like thing, given Shadow DOM is not too common.

In Gecko all these things are stored in `nsSlots`, in WebKit and Blink in `NodeRareData`.